File: Ruby/1010.rb
codigo, num, preco = gets.split
codigo = codigo.to_i
num = num.to_i
preco = preco.to_f

total = num * preco

codigo, num, preco = gets.split
codigo = codigo.to_i
num = num.to_i
preco = preco.to_f

total = total + num * preco
puts ("VALOR A PAGAR: R$ %0.2f" % total)

File: Lua/1010.lua
codigo = io.read("*n")
num = io.read("*n")
preco = io.read("*n")

total = num * preco

codigo = io.read("*n")
num = io.read("*n")
preco = io.read("*n")

total = total + num * preco

print(string.format("VALOR A PAGAR: R$ %.2f", total))